manifests/y/youxiao/yxfile/2.1.9.18/youxiao.yxfile.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: youxiao.yxfile
+PackageVersion: 2.1.9.18
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  Scope: user
```

```suggestion
  Scope: machine
```
`C:\Program Files (x86)\优效文件助手\yxfile.exe`